src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -112,7 +115,17 @@ private static void setupDrivers(Application application) {
 
         DriverFeature.updateDriverInfo();
     }
-
+    private static void setupDriverCountTest(Application application, DriverManager driverManager) {
+        application.addTest("Count Drivers", (ActionEvent e) -> {
+            DriverCounterVisitor counter = new DriverCounterVisitor();
+            for (Job2dDriver driver : driverManager.getAllDrivers()) {
+                if (driver instanceof VisitableDriver) {
+                    ((VisitableDriver) driver).accept(counter);
+                }
+            }
+            logger.info(String.valueOf(counter.getCount()));;
```

Comment:
Too much code. TestJobs2dApp should configure app functions, not implement them. Move to a separate class.

Reply:
done